Examples/Modules/gaussian_beam/PICMI_inputs_gaussian_beam.py

```diff
@@ -25,8 +25,10 @@
 grid = picmi.Cartesian3DGrid(number_of_cells = [nx, ny, nz],
                              lower_bound = [xmin, ymin, zmin],
                              upper_bound = [xmax, ymax, zmax],
-                             lower_boundary_conditions = ['periodic', 'periodic', 'open'],
-                             upper_boundary_conditions = ['periodic', 'periodic', 'open'],
+                             lower_boundary_conditions = ['periodic', 'periodic', 'pml'],
+                             upper_boundary_conditions = ['periodic', 'periodic', 'pml'],
+                             lower_boundary_conditions_particles = ['periodic', 'periodic', 'open'],
```

Comment:
For the particles, 'open' boundary has the specific meaning that particles are lost at the boundary but continue depositing in the PML layers. I don't think that this is fully implemented in WarpX. I would suggest using 'absorbing' instead.